tests/meta/report/test_sample_helper.py

```diff
@@ -0,0 +1 @@
+"""Tests the SampleHelper class"""
```

Comment:
What is the function of this one? Seems empty

Reply:
removed